packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/mocks.tsx

```diff
@@ -15,16 +21,94 @@ export const MOCK_RESET_TOKEN = 'mockResetToken';
 export const MOCK_RECOVERY_KEY_ID = 'mockRecoveryKeyId';
 export const MOCK_KB = 'mockkB';
 
+// TODO: combine a lot of mocks with AccountRecoveryResetPassword
```

Comment:
Yeah, I haven't gotten to the testing part of the architecture doc but definitely going to call out that we need to share more mocks 🙈 So seems fine for now.

Reply:
Right, the mocks for `CompleteResetPassword` and `AccountRecoveryConfirmKey` are almost identical too. Maybe they should be combined into the top level `ResetPassword` folder